chatbot-ui/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Use cursor to help to write some tests, also we want to test if a user types something in, do we get a response?